src/Tests/Consolonia.Gallery.Tests/FontsTests.cs

```diff
@@ -0,0 +1,67 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Avalonia.Controls;
+using Avalonia.Threading;
+using Consolonia.Gallery.Gallery.GalleryViews;
+using Consolonia.Gallery.Tests.Base;
+using Consolonia.NUnit;
+using NUnit.Framework;
+
+namespace Consolonia.Gallery.Tests
+{
+    /// <summary>
+    ///     Unit test for TextBlock view
+    /// </summary>
+    [TestFixture]
+    internal class FontsTests : GalleryTestsBaseBase
+    {
+        [Test]
+        public async Task DisplaysBasicText()
+        {
+            await SelectFont("ConsoleDefault");
+            await UITest.AssertHasText("Hello World!");
+        }
+
+        [Test]
+        public async Task DisplaysWideTermText()
+        {
+            await SelectFont("WideTerm");
+            await UITest.AssertHasRawText("Ｈ️ｅ️ｌ️ｌ️ｏ️  Ｗ️ｏ️ｒ️ｌ️ｄ️！️");
+        }
+
+        [Test]
+        public async Task DisplaysBrailleText()
+        {
+            await SelectFont("Braille");
+
+            await UITest.AssertHasRawText(
+                "⣇⣸ ⢀⡀ ⡇ ⡇ ⢀⡀   ⡇⢸ ⢀⡀ ⡀⣀ ⡇ ⢀⣸ ⡇",
+                "⠇⠸ ⠣⠭ ⠣ ⠣ ⠣⠜   ⠟⠻ ⠣⠜ ⠏  ⠣ ⠣⠼ ⠅");
+        }
+
+        [Test]
+        public async Task DisplaysCacaFontText()
+        {
+            await SelectFont("Doom");
+            await UITest.AssertHasRawText(
+                @" _   _       _ _         _    _             _     _ ",
+                @"| | | |     | | |       | |  | |           | |   | |",
+                @"| |_| | ___ | | | ___   | |  | |  ___  _ __| | __| |",
+                @"|  _  |/ _ \| | |/ _ \  | |/\| | / _ \| '__| |/ _` |",
+                @"| | | |  __/| | | (_) | \  /\  /| (_) | |  | | (_| |",
+                @"\_| |_/\___||_|_|\___/   \/  \/  \___/|_|  |_|\__,_|"
+            );
+        }
+
+        private static async Task SelectFont(string font)
```

Comment:
Why can't we select the font using regular e2e approach as we use everywhere?

Reply:
I spent about 4 hours fucking around with sending Tab, down arrow, etc. and I just couldn't get it to work. The focus was going someplace weird.